--- acinclude.m4.in.orig	2005-06-26 14:38:05.000000000 +0200
+++ acinclude.m4.in	2005-06-26 14:39:13.000000000 +0200
@@ -443,7 +443,7 @@
     kde_datadir='\${datadir}/apps'
   fi
   if test -z "$kde_locale"; then
-    kde_locale='\${datadir}/locale'
+    kde_locale='\${prefix}/share/locale'
   fi
   if test -z "$kde_cgidir"; then
     kde_cgidir='\${exec_prefix}/cgi-bin'
@@ -531,17 +531,12 @@
 
 AC_DEFUN([KDE_MISSING_PROG_ERROR],
 [
-    AC_MSG_ERROR([The important program $1 was not found!
-Please check whether you installed KDE correctly.
-])
+	have_kde="no"
 ])
 
 AC_DEFUN([KDE_MISSING_ARTS_ERROR],
 [
-    AC_MSG_ERROR([The important program $1 was not found!
-Please check whether you installed aRts correctly or use
---without-arts to compile without aRts support (this will remove functionality).
-])
+	have_kde="no"
 ])
 
 AC_DEFUN([KDE_SET_DEFAULT_BINDIRS],
@@ -590,7 +585,7 @@
               if test "$kde_qtsubver" -le 1; then
                 kde32ornewer=
               fi
-              if test "$kde_qtsubver" -le 2 -o `$KDECONFIG --version | grep KDE | sed 's/KDE: \(...\).*/\1/'` = 3.2; then
+              if test "$kde_qtsubver" -le 2; then
                 kde33ornewer=
               fi
             fi
@@ -1634,15 +1629,7 @@
 
 AC_MSG_RESULT([$kde_cv_uic_plugins])
 if test "$kde_cv_uic_plugins" != yes; then
-	AC_MSG_ERROR([
-you need to install kdelibs first.
-
-If you did install kdelibs, then the Qt version that is picked up by
-this configure is not the same version you used to compile kdelibs. 
-The Qt Plugin installed by kdelibs is *ONLY* loadable if its the 
-same Qt version, compiled with the same compiler and the same Qt
-configuration settings.
-])
+	have_kde="no"
 fi
 fi
 ])
@@ -1791,11 +1778,8 @@
 ac_kde_includes="$kde_incdir"
 
 if test -n "$ac_kde_includes" && test ! -r "$ac_kde_includes/$kde_check_header"; then
-  AC_MSG_ERROR([
-in the prefix, you've chosen, are no KDE headers installed. This will fail.
-So, check this please and use another prefix!])
+	have_kde="no"
 fi
-
 kde_libdirs="$kde_libs_prefix/lib${kdelibsuff} /usr/lib/kde/lib${kdelibsuff} /usr/local/kde/lib${kdelibsuff} /usr/kde/lib${kdelibsuff} /usr/lib${kdelibsuff}/kde /usr/lib${kdelibsuff}/kde3 /usr/lib${kdelibsuff} /usr/X11R6/lib${kdelibsuff} /usr/local/lib${kdelibsuff} /opt/kde3/lib${kdelibsuff} /opt/kde/lib${kdelibsuff} /usr/X11R6/kde/lib${kdelibsuff}"
 test -n "$KDEDIR" && kde_libdirs="$KDEDIR/lib${kdelibsuff} $KDEDIR $kde_libdirs"
 kde_libdirs="$ac_kde_libraries $libdir $kde_libdirs"
@@ -1807,15 +1791,11 @@
 AC_FIND_FILE("kde3/plugins/designer/kdewidgets.la", $kde_libdirs, kde_widgetdir)
 
 if test -n "$ac_kde_libraries" && test ! -r "$ac_kde_libraries/$kde_check_lib"; then
-AC_MSG_ERROR([
-in the prefix, you've chosen, are no KDE libraries installed. This will fail.
-So, check this please and use another prefix!])
+	have_kde="no"
 fi
 
 if test -n "$kde_widgetdir" && test ! -r "$kde_widgetdir/kde3/plugins/designer/kdewidgets.la"; then
-AC_MSG_ERROR([
-I can't find the designer plugins. These are required and should have been installed
-by kdelibs])
+	have_kde="no"
 fi
 
 if test -n "$kde_widgetdir"; then
@@ -1892,7 +1872,9 @@
 AC_SUBST(all_libraries)
 
 if test -z "$1"; then
-KDE_CHECK_UIC_PLUGINS
+	if test "$have_kde" != "no"; then
+		KDE_CHECK_UIC_PLUGINS
+	fi
 fi
 
 ac_kde_libraries="$kde_libdir"
@@ -2945,6 +2927,7 @@
 
 ])
 
+
 dnl AC_VALIDIFY_CXXFLAGS checks for forbidden flags the user may have given
 AC_DEFUN([AC_VALIDIFY_CXXFLAGS],
 [dnl
@@ -4808,21 +4791,18 @@
   KDE_SET_DEFAULT_BINDIRS
   if test "x$prefix" = "xNONE"; then
     dnl no prefix given: look for kde-config in the PATH and deduce the prefix from it
-    KDE_FIND_PATH(kde-config, KDECONFIG, [$kde_default_bindirs], [KDE_MISSING_PROG_ERROR(kde-config)], [], prepend)
+    KDE_FIND_PATH(kde-config, KDECONFIG, [$kde_default_bindirs], [], [], prepend)
   else
     dnl prefix given: look for kde-config, preferrably in prefix, otherwise in PATH
     kde_save_PATH="$PATH"
     PATH="$exec_prefix/bin:$prefix/bin:$PATH"
-    KDE_FIND_PATH(kde-config, KDECONFIG, [$kde_default_bindirs], [KDE_MISSING_PROG_ERROR(kde-config)], [], prepend)
+    KDE_FIND_PATH(kde-config, KDECONFIG, [$kde_default_bindirs], [], [], prepend)
     PATH="$kde_save_PATH"
   fi
 
   kde_libs_prefix=`$KDECONFIG --prefix`
   if test -z "$kde_libs_prefix" || test ! -x "$kde_libs_prefix"; then
-       AC_MSG_ERROR([$KDECONFIG --prefix outputed the non existant prefix '$kde_libs_prefix' for kdelibs.
-                    This means it has been moved since you installed it.
-                    This won't work. Please recompile kdelibs for the new prefix.
-                    ])
+  	have_kde="no"
   fi
   kde_libs_htmldir=`$KDECONFIG --install html --expandvars`
   kde_libs_suffix=`$KDECONFIG --libsuffix`
